fe/fe-core/src/main/java/org/apache/doris/datasource/property/storage/MinioProperties.java

```diff
@@ -75,4 +75,18 @@ public static boolean guessIsMe(Map<String, String> origProps) {
     protected Set<Pattern> endpointPatterns() {
         return ImmutableSet.of(Pattern.compile("^(?:https?://)?[a-zA-Z0-9.-]+(?::\\d+)?$"));
     }
+
+    @Override
+    public void initializeHadoopStorageConfig() {
+        hadoopStorageConfig = new Configuration();
+        hadoopStorageConfig.set("fs.s3.impl", "org.apache.hadoop.fs.s3a.S3AFileSystem");
```

Comment:
Do we need hadoop prop to access minio?

Reply:
We have users who opt to use MinIO as their storage solution. MinIO generally follows the s3 URI schema.